lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -58,17 +59,22 @@ def self.canonicalize_url(driver_url, config)
 
     # Load balancer methods
     def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
-      security_group_name = lb_options[:security_group_name] || 'default'
+      security_group_name = lb_options[:security_group_name]
```

Comment:
Question: what happened to security_group_name defaulting to `'default'`?  Is that not needed anymore?